sshd.c

```diff
@@ -884,16 +897,16 @@ privsep_postauth(Authctxt *authctxt)
 		    posix_spawn_file_actions_adddup2(&actions, tmp_sock, STDIN_FILENO) != 0 ||
 		    posix_spawn_file_actions_adddup2(&actions, tmp_sock, STDOUT_FILENO) != 0 ||
 		    posix_spawn_file_actions_adddup2(&actions, pmonitor->m_recvfd, PRIVSEP_MONITOR_FD) != 0)
-			error("posix_spawn initialization failed");
-		else {
+			fatal("posix_spawn initialization failed");
```

Comment:
since we exit the process, I am assuming we have sufficient log messages to debug the issue in production?

Reply:
Not really, the above functions are fairly lightweight and should not fail in production scenarios, fatal here is catch any broken assumptions at implementation time